### ISSUE TITLE
Yuma Inaura was Here 2019-05-25 on Twitter

### ISSUE THREAD

**Issue by YumaInaura** [YumaInaura/YumaInaura#2007]
# YumaInaura Second

@yumainaura is my brother

![image](http://pbs.twimg.com/profile_images/1117366091192291333/IXEjoF1f_400x400.jpg)

# What kind of cash principle that a person can not be admitted if it is accepted or depressed So always go against my senses. When it's floating it's denting itself, and when it's depressed it's going to raise it. 2019-05-24 on Twitter
 [*](https://twitter.com/YumaInaura/status/1131938422971555842")

>人に認められるとはしゃぎ出し認められないと落ち込むなんてどんな現金主義。だから常に自分の感覚に逆行するんだ。浮かれているときは自分を凹ませ落ち込んでいるときは逆に高める。 2019-05-24 on Twitter
>https://t.co/k85HkSDXEO
# Finally I encountered a taxi that can not use a credit card.
 [*](https://twitter.com/YumaInaura/status/1131949036297056256")
![image](https://pbs.twimg.com/media/D7UBy7SU0AA3No7.jpg)
![image](https://pbs.twimg.com/media/D7UBy7PVUAI_hpD.jpg)
[@YumaInaura](https://twitter.com/YumaInaura/)

>ついにクレジットカードが使えないタクシーに遭遇した。 https://t.co/eFyDZICSKe
![image](https://pbs.twimg.com/media/D7UBy7SU0AA3No7.jpg)

![image](https://pbs.twimg.com/media/D7UBy7PVUAI_hpD.jpg)

# Many Lawson people came to give me a towel.
 [*](https://twitter.com/YumaInaura/status/1132003100745584640")
![image](https://pbs.twimg.com/media/D7UzI0GUYAA-2PK.jpg)
[@YumaInaura](https://twitter.com/YumaInaura/)

>ローソンの人がたくさんおしぼりをくれるようになった。 https://t.co/QtJ1OSWbJP
![image](https://pbs.twimg.com/media/D7UzI0GUYAA-2PK.jpg)

# The gap of the menu contrary to the name of restaurant Paul Wright.
 [*](https://twitter.com/YumaInaura/status/1132003353012006912")
![image](https://pbs.twimg.com/media/D7UzV-RUcAAoHtn.jpg)
[@YumaInaura](https://twitter.com/YumaInaura/)

>レストランポールライトという名前に反したメニューのギャップ。 https://t.co/kYwLCHxcvC
![image](https://pbs.twimg.com/media/D7UzV-RUcAAoHtn.jpg)

# After I went to the hospital my mother provokes with family LINE.
 [*](https://twitter.com/YumaInaura/status/1132004610204246016")
![image](https://pbs.twimg.com/media/D7U0bMuUwAAz-UD.jpg)
[@YumaInaura](https://twitter.com/YumaInaura/)

>僕が病院に行った後に家族LINEで母が挑発してくる。 https://t.co/VZO1laadro
![image](https://pbs.twimg.com/media/D7U0bMuUwAAz-UD.jpg)

# It takes 30 seconds to notice that you are really sad.
 [*](https://twitter.com/YumaInaura/status/1132006877556903936")
[@YumaInaura](https://twitter.com/YumaInaura/)

>あなたが本当に悲しくなったことに気づくのは30秒後。
# The magic of naming something.
 [*](https://twitter.com/YumaInaura/status/1132012159875993600")
![image](https://pbs.twimg.com/media/D7U7V9iU0AAC_q5.jpg)
[@YumaInaura](https://twitter.com/YumaInaura/)

>なんかネーミングの魔術。 https://t.co/kwPQ1vVk7l
![image](https://pbs.twimg.com/media/D7U7V9iU0AAC_q5.jpg)

# I thought it was an amusement park.
 [*](https://twitter.com/YumaInaura/status/1132013416493674496")
[@s11rickey](https://twitter.com/s11rickey/)

>遊園地かと思った。 https://t.co/R7w1vCWqXA
# Thirty kinds of jams interfere with selection, but six kinds of jams seem effective.
 [*](https://twitter.com/YumaInaura/status/1132016439752183808")
[@shupeiman](https://twitter.com/shupeiman/)

>三十種類のジャムは選択を妨げるが六種類のジャムは有効らしい。 https://t.co/e4NqjfZK2w
# The biggest challenge is how to bite the weight of money when working with fixed pay?
 [*](https://twitter.com/YumaInaura/status/1132035310798364672")
[@YumaInaura](https://twitter.com/YumaInaura/)

>固定給で働くとお金の重さをどうやって噛みしめるかが最大の課題では？
# After I went to the hospital my mother provokes with family LINE. 2019-05-24 on Twitter
 [*](https://twitter.com/YumaInaura/status/1132059975876980736")

>僕が病院に行った後に家族LINEで母が挑発してくる。  2019-05-24 on Twitter
>https://t.co/9iz6dq0EyB
# Yuma Inaura was here 2019-05-24 on Twitter (Yuma Inaura YumaInaura)
 [*](https://twitter.com/YumaInaura/status/1132076084206292999")

>いなうらゆうま はここにいた 2019-05-24 on Twitter ( 稲浦悠馬 YumaInaura )
>https://t.co/VKlvJZeqce
# Yuma Inaura was Here 2019-05-24 on Twitter
 [*](https://twitter.com/YumaInaura/status/1132076085267460096")

>Yuma Inaura was Here 2019-05-24 on Twitter
>https://t.co/9h9sLGLRl8
# There is a remote work trial of a Rails engineer 2019-05-24 on Twitter [poem] on @Qiita
 [*](https://twitter.com/YumaInaura/status/1132091436579483648")

>とあるRailsエンジニアのリモートワークトライアル 2019-05-24 on Twitter [ポエム] on @Qiita https://t.co/I122AH79o9
# Hello.
 [*](https://twitter.com/YumaInaura/status/1132186811697025024")
![image](https://pbs.twimg.com/media/D7XaK4VU0AA3FGD.jpg)
[@YumaInaura](https://twitter.com/YumaInaura/)

>こんにちは。 https://t.co/cBMZgEnvuw
![image](https://pbs.twimg.com/media/D7XaK4VU0AA3FGD.jpg)

# This is a whimsical pudding from Starbucks. Is it really fancy?
 [*](https://twitter.com/YumaInaura/status/1132186812980449280")
![image](https://pbs.twimg.com/media/D7XaRyxUIAAHCtq.jpg)
[@YumaInaura](https://twitter.com/YumaInaura/)

>スタバにまさかの気まぐれプリン。本当に気まぐれなのかな？ https://t.co/ex0maRmUQl
![image](https://pbs.twimg.com/media/D7XaRyxUIAAHCtq.jpg)

# Standing desk in the corner of the room.

The loading books are still open and packing furniture is open.
 [*](https://twitter.com/YumaInaura/status/1132187314182971393")
![image](https://pbs.twimg.com/media/D7Xai7MUwAA-p8b.jpg)
[@YumaInaura](https://twitter.com/YumaInaura/)

>部屋の片隅にスタンディングデスク。
>
>積み本はまだしも梱包開かずの積み家具はヤバイ。 https://t.co/wQqm3lA8jg
![image](https://pbs.twimg.com/media/D7Xai7MUwAA-p8b.jpg)

# Because there are only 2 outlets at the corner of Starbucks, I share it with others. One is always open for someone, but the keyboard is rather noisy and people tend to sit down, so I run away soon.
 [*](https://twitter.com/YumaInaura/status/1132187566940205056")
![image](https://pbs.twimg.com/media/D7Xa8oIUEAAOw9V.jpg)
[@YumaInaura](https://twitter.com/YumaInaura/)

>スタバの角にたった2個のコンセントがあるから仲良くシェアする。いつもひとつは誰かのために空けてげてるけど、わりとキーボードがうるさい人とかが座りがちなので逆にすぐ逃げる。 https://t.co/m1JB4LyuxP
![image](https://pbs.twimg.com/media/D7Xa8oIUEAAOw9V.jpg)

# Which way do you like good morning and good night?
 [*](https://twitter.com/YumaInaura/status/1132188824665116672")
[@YumaInaura](https://twitter.com/YumaInaura/)

>おはようとおやすみどっちが好きなの？
# If the programmer who can do live coding is searching for Google and Google, it may be easy to tell the actual situation.
 [*](https://twitter.com/YumaInaura/status/1132189577085456384")
[@xhack20](https://twitter.com/xhack20/)

>ライブコーディングでもできるプログラマがガリガリとGoogle検索したら実情が伝わりやすいのかも。 https://t.co/gVQy5Rg0i9
# An agile person said waterfall actually seems to make many turns.
 [*](https://twitter.com/YumaInaura/status/1132189826948640768")
[@kyiwj5eppdc1o01](https://twitter.com/kyiwj5eppdc1o01/)

>ウォーターフォールって実は何回転もするらしいってアジャイルの人が言ってた。 https://t.co/V7qaKqtUY1
# By the way, I mute the mute function of tweet unit.
 [*](https://twitter.com/YumaInaura/status/1132190082251681793")
[@YumaInaura](https://twitter.com/YumaInaura/)

>ちなみにツイート単位のミュート機能をかましてます。
# I spread the tablet at Starbucks and I play with a game right next to me. I felt uncomfortable and I went out in about 5 minutes.
 [*](https://twitter.com/YumaInaura/status/1132190835812945920")
[@YumaInaura](https://twitter.com/YumaInaura/)

>スタバでタブレットを広げて僕のすぐ隣でゲームで遊んでるおっちゃんの指使いが目に触って視界が辛かったんだけどこの店なんか違う？と居心地の悪さを感じたのかおっちゃんは5分ぐらいで出て行った。
# Even if it is an engineer who has the same ability, the image is likely to change twice as much as the unit price simply because the company's sales and bargaining power of the contract are different.
 [*](https://twitter.com/YumaInaura/status/1132191338521251841")
[@xhack20](https://twitter.com/xhack20/)

>仮に同じ能力を持っているエンジニアでもそもそも会社の売上や契約の交渉力が違うだけで単価なんて軽く2倍は変わりそうなイメージ。 https://t.co/Q5AVU9wSQi
# I tend to choose cold pita.
 [*](https://twitter.com/YumaInaura/status/1132191590250819585")
[@rails_java_like](https://twitter.com/rails_java_like/)

>冷えピタを選んでしまいがち。 https://t.co/lNbbKFZ9eD
# Birth of CSS Engineer.
 [*](https://twitter.com/YumaInaura/status/1132191839832817666")
[@ginza_no_motcii](https://twitter.com/ginza_no_motcii/)

>CSSエンジニアの誕生。 https://t.co/6QagIlo38X
# An unexperienced hunter who kills the prey.
 [*](https://twitter.com/YumaInaura/status/1132192348891279362")
[@shiraponsu](https://twitter.com/shiraponsu/)

>未経験でも獲物を仕留めたハンター。 https://t.co/SWVnmOAvmJ
# Isn't there a way to say that the smartphone isn't completely cut off, either?
 [*](https://twitter.com/YumaInaura/status/1132192350296465410")
[@oumi2_jun](https://twitter.com/oumi2_jun/)

>スマホ完全断捨離じゃなくてガラケーっていう手段もあるのか。 https://t.co/dPmoAAVyzD
# The ticket has already been sold out and it has a great momentum.
 [*](https://twitter.com/YumaInaura/status/1132192851293376513")
[@_ikeshin](https://twitter.com/_ikeshin/)

>既にチケット売り切れててすごい勢い。 https://t.co/DLDLaFlkms
# Do you want to make the best effort when you want to get rid of it?
 [*](https://twitter.com/YumaInaura/status/1132192852736274432")
[@YumaInaura](https://twitter.com/YumaInaura/)

>一番気を抜きたいときに一番努力したくないか？
# Stop the trouble report.
 [*](https://twitter.com/YumaInaura/status/1132193350856003584")
[@nu4hlthkouoxdss](https://twitter.com/nu4hlthkouoxdss/)

>障害報告やめて。 https://t.co/rUq68cVtT7
# Fast delivery. Maybe agile developing domino pizza.
 [*](https://twitter.com/YumaInaura/status/1132193609480990720")
[@pickles_works](https://twitter.com/pickles_works/)

>迅速なデリバリー。たぶんアジャイル開発なドミノピザ。 https://t.co/3HT4QnKDgT
# It is not until you seek the price that you can see your worth.
 [*](https://twitter.com/YumaInaura/status/1132193610697302016")
[@pickles_works](https://twitter.com/pickles_works/)

>対価を求めてみて初めて自分の価値が分かる。 https://t.co/yUes7BiwHX
# Nature-friendly mind to clean the wiring as much as possible while charging the MacBook and not to spoil the view of the wooden table of the Starbucks.
 [*](https://twitter.com/YumaInaura/status/1132201154786385920")
![image](https://pbs.twimg.com/media/D7XnPhaU8AAOh-9.jpg)
![image](https://pbs.twimg.com/media/D7XnPhSUYAEXMkd.jpg)
![image](https://pbs.twimg.com/media/D7XnPhUUEAASFDX.jpg)
[@YumaInaura](https://twitter.com/YumaInaura/)

>MacBookの充電中もなるべく配線を綺麗にしてスタバの木のテーブルの景観を損なわないための自然愛護的なこころみ。 https://t.co/RP1zH2DLi1
![image](https://pbs.twimg.com/media/D7XnPhaU8AAOh-9.jpg)

![image](https://pbs.twimg.com/media/D7XnPhSUYAEXMkd.jpg)

![image](https://pbs.twimg.com/media/D7XnPhUUEAASFDX.jpg)

# x In this society
o  at this company
 [*](https://twitter.com/YumaInaura/status/1132201654059655168")
[@vacanada](https://twitter.com/vacanada/)

>x この社会において
>o この会社において https://t.co/8Do8KIbFYK
# Even if I think for 3 seconds, I didn't get an answer.
 [*](https://twitter.com/YumaInaura/status/1132201909559758849")
[@ihcomega](https://twitter.com/ihcomega/)

>3秒考えても答えが出なかったのでたぶん永遠に出ない。 https://t.co/0E9VHVvkWD
# Sound movement.
 [*](https://twitter.com/YumaInaura/status/1132202662777069568")
[@blogucci](https://twitter.com/blogucci/)

>健全な動き。 https://t.co/vxP5c1FzCV
# My BEHAVIOR who runs through who I like or hate is the first to say first:
 [*](https://twitter.com/YumaInaura/status/1132202914351443968")
[@_marony](https://twitter.com/_marony/)

>好きとか嫌いとか最初に言いだしたのは誰なのかしら駆け抜けてゆく私の BEHAVIOR! https://t.co/4AXs1Gcbl3
# Cm7
 [*](https://twitter.com/YumaInaura/status/1132220783705579521")
[@Seiver17808307](https://twitter.com/Seiver17808307/)

>Cm7 https://t.co/nk3nrFanhq
# Is it because of the annual income?
 [*](https://twitter.com/YumaInaura/status/1132220784942784512")
[@kazukundayo](https://twitter.com/kazukundayo/)

>年収のせいなのかな？ https://t.co/LgJDRjPYbT
# Freelance seems to be life normal mode.
 [*](https://twitter.com/YumaInaura/status/1132221034860384257")
[@Seiver17808307](https://twitter.com/Seiver17808307/)

>フリーランスは人生ノーマルモードらしい。 https://t.co/NgPMctIXBc
# Like Twitter, there are a lot of exchanges of replies, but I'm sure I'm glad "a clear fan letter".
 [*](https://twitter.com/YumaInaura/status/1132221285872726017")
[@lulun_ADHD](https://twitter.com/lulun_ADHD/)

>Twitterではいいねとか、リプライのやりとりは多いけど「明確なファンレター」って確かに嬉しいだろうな。 https://t.co/dzhR2crmIf
# Physical condition management is not 99.9% of work recently? I felt that.
 [*](https://twitter.com/YumaInaura/status/1132221538466275328")
[@nicoberu12](https://twitter.com/nicoberu12/)

>最近は体調管理が仕事の99.9%ではないか？という気がしてきた。 https://t.co/wH7sxNjcbR
# Overtime work time is 160 hours and productivity seems to become about 1/4.
 [*](https://twitter.com/YumaInaura/status/1132222797856403461")
[@YumaInaura](https://twitter.com/YumaInaura/)

>残業時間が160時間って生産性は1/4ぐらいになってそう。
# Somehow we have a strong difference between success and failure frames.
 [*](https://twitter.com/YumaInaura/status/1132222798934364161")
[@Mck_0109](https://twitter.com/Mck_0109/)

>なぜか成功と失敗というフレームの違いを僕らは強烈に持ってるよね。 https://t.co/eKepeLVKhC
# Haruki Murakami says, "Everything is the same." Is Haruki Murakami's "Everything is so."
 [*](https://twitter.com/YumaInaura/status/1132223302456373249")
[@EssentialWord1](https://twitter.com/EssentialWord1/)

>村上春樹がいう「なんでもそうだけど」はあくまで村上春樹の「なんでもそうだけど」。 https://t.co/V0nV5n7wny
# Partner thinking sometimes returns with a flat point of view, but it quickly escapes tens of thousands of times as the legacy master-slave relationship creeps in without sound.
 [*](https://twitter.com/YumaInaura/status/1132223805193986048")
[@chikagoo](https://twitter.com/chikagoo/)

>パートナー思考って、たまにフラットな視点で立ち戻るけれど、すぐにレガシーな主従関係目線が音もなく忍び寄るので、何万回でも脱出しい。 https://t.co/Kn2bfpaj14
# A guy who is embarrassed to hit a hit.
 [*](https://twitter.com/YumaInaura/status/1132227324684845056")
[@YumaInaura](https://twitter.com/YumaInaura/)

>ヒットを打てるのに恥ずかしがって打たないやつ。
# Determine the difference between "The file content exists" and "Empty file or file does not exist" in shell (-s filepath) on @Qiita
 [*](https://twitter.com/YumaInaura/status/1132230094427004928")

>#shell で「ファイルの中身が存在する」「空ファイル or ファイルが存在しない」の違いを判定する ( -s filepath ) on @Qiita https://t.co/jzKcXLuRbZ
# Want to.
 [*](https://twitter.com/YumaInaura/status/1132236637117734912")
[@YumaInaura](https://twitter.com/YumaInaura/)

>したい。
# My business card Qiita

Engineer to meditate (by INAURA ま 稲 @yumainaura) — suspicion of “mindfulness” learning from Google and Steve Jobs — all programmers SE must-see! -Qiita
 [*](https://twitter.com/YumaInaura/status/1132239912076341248")

>僕の名刺Qiita
>
>瞑想するエンジニア ( by いなうらゆうま 稲浦悠馬 @yumainaura ) — Googleやスティーブ・ジョブズに学ぶ「マインドフルネス」のススメ— 全プログラマ SE 必見！ - Qiita https://t.co/VOyCaksI20
# Yogurt that sweet jelly has accumulated in the lower part and is mixed by oneself.
 [*](https://twitter.com/YumaInaura/status/1132241167423160320")
![image](https://pbs.twimg.com/media/D7YLlohV4AAVj_y.jpg)
[@YumaInaura](https://twitter.com/YumaInaura/)

>下の方に甘いゼリーが溜まっていて自分で混ぜられるヨーグルト。 https://t.co/K13O5E12gf
![image](https://pbs.twimg.com/media/D7YLlohV4AAVj_y.jpg)

# Store name to remember someone on Twitter.
 [*](https://twitter.com/YumaInaura/status/1132241168589131778")
![image](https://pbs.twimg.com/media/D7YLsPcUwAEQuRL.jpg)
[@YumaInaura](https://twitter.com/YumaInaura/)

>Twitterの誰かを思い出す店名。 https://t.co/sbIim7ylHW
![image](https://pbs.twimg.com/media/D7YLsPcUwAEQuRL.jpg)

# I tried to make it a dot picture.
 [*](https://twitter.com/YumaInaura/status/1132241419580497920")
![image](https://pbs.twimg.com/media/D7YL4PJU0AAurAM.jpg)
[@YumaInaura](https://twitter.com/YumaInaura/)

>ドット絵になってみた。 https://t.co/M1e0nn4jjp
![image](https://pbs.twimg.com/media/D7YL4PJU0AAurAM.jpg)

# It protects properly when taking a shot with PayPay.
 [*](https://twitter.com/YumaInaura/status/1132242175037566976")
![image](https://pbs.twimg.com/media/D7YMfIAVsAAn4aE.jpg)
[@YumaInaura](https://twitter.com/YumaInaura/)

>PayPayでスクショを撮ったときにちゃんと保護してくれる。 https://t.co/PvFbxjYzGJ
![image](https://pbs.twimg.com/media/D7YMfIAVsAAn4aE.jpg)

# Are you working hard to earn trust one year from now? Two years later? Three years later?
 [*](https://twitter.com/YumaInaura/status/1132245443281186817")
[@YumaInaura](https://twitter.com/YumaInaura/)

>1年後も信頼獲得に努力できている？2年後も？3年後も？
# Trust acquisition should be a natural practice to ride a bicycle if it is a skill.
 [*](https://twitter.com/YumaInaura/status/1132245702543732736")
[@YumaInaura](https://twitter.com/YumaInaura/)

>信頼獲得もスキルなら自転車に乗るように自然な習慣にできるはず。
# People who are rooted in the act of gaining trust are indeed in reality.
 [*](https://twitter.com/YumaInaura/status/1132245703625830401")
[@YumaInaura](https://twitter.com/YumaInaura/)

>信頼獲得行動が根に身についている人は確かに現実にいる。
# Use threads in Slack or write to main
It is certain that things like the level going out will change
Anyway, I'm too careful
I think too much thinking about me
Volume 2
 [*](https://twitter.com/YumaInaura/status/1132252742460686336")
[@YumaInaura](https://twitter.com/YumaInaura/)

>Slackでスレッドを使うかメインに書くかでも
>出しゃばってるレベルのようなものが変化するのは確実で
>とにかく気をつかいすぎるのは
>どう考えても考えすぎな僕が悪い
>第2巻
# It would be nice to have a chatty person in the team. Whether it's a private or work community.
 [*](https://twitter.com/YumaInaura/status/1132252997075890179")
[@YumaInaura](https://twitter.com/YumaInaura/)

>おしゃべりな人がチームに1人はいると良いよね。プライベートのコミュニティか仕事のコミュニティかに関わらず。
# Even strange healing to the world.
 [*](https://twitter.com/YumaInaura/status/1132253248075657216")
[@kenjiro_isomura](https://twitter.com/kenjiro_isomura/)

>世にも奇妙なヒーリング。 https://t.co/Xf4JMwWnPB
# x In this society 2019-05-25 on Twitter
 [*](https://twitter.com/YumaInaura/status/1132256267857158147")

>x この社会において 2019-05-25 on Twitter
>https://t.co/QyLNM8yDra
# Combine high sense of security and sense of responsibility.
 [*](https://twitter.com/YumaInaura/status/1132256773102956546")
[@YumaInaura](https://twitter.com/YumaInaura/)

>安心感と責任感が高レベル結合せよ。
# Did you find an ideal person from that?
 [*](https://twitter.com/YumaInaura/status/1132274892668170241")
![image](https://pbs.twimg.com/media/D7YqQiaUEAAWBFV.jpg)
[@YumaInaura](https://twitter.com/YumaInaura/)

>あれから理想の人見つかりました？ https://t.co/Xr7fR2Z2ai
![image](https://pbs.twimg.com/media/D7YqQiaUEAAWBFV.jpg)

# Your heart is not black, right? That's ok. No problem.
 [*](https://twitter.com/YumaInaura/status/1132274893922287616")
![image](https://pbs.twimg.com/media/D7YqXwVVsAACZLu.jpg)
[@YumaInaura](https://twitter.com/YumaInaura/)

>心は黒ずんでませんよね？大丈夫だ問題ない。 https://t.co/b0WN3jQM8y
![image](https://pbs.twimg.com/media/D7YqXwVVsAACZLu.jpg)

# Popular because there is always only an ultra-low-calorie pointed green smoothie.
 [*](https://twitter.com/YumaInaura/status/1132275142749384704")
![image](https://pbs.twimg.com/media/D7YqgygUwAAnbeR.jpg)
[@YumaInaura](https://twitter.com/YumaInaura/)

>超低カロリーの尖ったグリーンスムージーだけいつもないから人気者。 https://t.co/fYF6MYulMl
![image](https://pbs.twimg.com/media/D7YqgygUwAAnbeR.jpg)

# A popular concept of the pointed concept  because protein yogurt drinks are not always the only one
 [*](https://twitter.com/YumaInaura/status/1132275143995080710")
![image](https://pbs.twimg.com/media/D7Yqn5WV4AAGxT_.jpg)
[@YumaInaura](https://twitter.com/YumaInaura/)

>プロテインのヨーグルトドリンクだけいつもないから尖ったコンセプトの人気者 https://t.co/jsPwjEFdV5
![image](https://pbs.twimg.com/media/D7Yqn5WV4AAGxT_.jpg)

# As long as I have a sense of responsibility, I can concentrate on giving the final output regardless of the time unit to work with fixed pay.
 [*](https://twitter.com/YumaInaura/status/1132276398628802560")
[@YumaInaura](https://twitter.com/YumaInaura/)

>固定給で働くのは責任感さえあれば時間単位にとらわれず最終アウトプットを出すことに集中できるかな。
# Good luck Our aiming.

The latest work of the "Logless" series, "Logless Story (Stories)" is announced. CBT for Android devices will also be held, recruiting participants  via @ 4GamerNews
 [*](https://twitter.com/YumaInaura/status/1132281687222542336")

>がんばれ僕らのAiming。
>
>「ログレス」シリーズの最新作「ログレス物語（ストーリーズ）」が発表に。Android端末を対象としたCBTも開催決定，参加者を募集中 https://t.co/K4nxR6tSne via @4GamerNews
# Is there pressure to make you laugh at your nose?
 [*](https://twitter.com/YumaInaura/status/1132282189557555200")
[@YumaInaura](https://twitter.com/YumaInaura/)

>鼻で笑えるようなプレッシャーはあるかな？
# Watercolor
 [*](https://twitter.com/YumaInaura/status/1132282692639133697")
[@nabetani](https://twitter.com/nabetani/)

>水彩画 https://t.co/Os0x9WbDyQ
# Can you not switch between sense of responsibility and sense of irresponsibility?
 [*](https://twitter.com/YumaInaura/status/1132284205793079298")
[@YumaInaura](https://twitter.com/YumaInaura/)

>責任感と無責任感のスイッチングができないの？
# A sense of responsibility is necessary but excessive pressure is a dilemma that lowers the ability to perform tasks.
 [*](https://twitter.com/YumaInaura/status/1132284206942380032")
[@YumaInaura](https://twitter.com/YumaInaura/)

>責任感は大いに必要だが過度なプレッシャーは業務遂行能力を下げるジレンマ。
# Switching between sense of responsibility and sense of independence in personal rest.
 [*](https://twitter.com/YumaInaura/status/1132284454091730944")
[@YumaInaura](https://twitter.com/YumaInaura/)

>業務的責任感と個人的休息におけるいわば無責任感のスイッチング。
# There is a huge task every day, and somehow I go against entropy and keep my life by continuing to solve it.
 [*](https://twitter.com/YumaInaura/status/1132284954862280704")
[@YumaInaura](https://twitter.com/YumaInaura/)

>毎日巨大な課題があり解決し続けることによってどうにかエントロピーに逆行して命を保ってる。
# I have to make a mini commander in my head. I'm a soldier.
 [*](https://twitter.com/YumaInaura/status/1132285210161057793")
[@YumaInaura](https://twitter.com/YumaInaura/)

>頭の中にミニ司令官を作らないと。俺はソルジャーだし。
# What is the true rest?
 [*](https://twitter.com/YumaInaura/status/1132285459835416576")
[@YumaInaura](https://twitter.com/YumaInaura/)

>真実の休息とは何？
# My business card Qiita 2019-05-25 on Twitter
 [*](https://twitter.com/YumaInaura/status/1132285964544438273")

>僕の名刺Qiita 2019-05-25 on Twitter
>https://t.co/kZwHqVmzwe
# "boot
Boots (boots)
There is no meaning of activation. It seems that there are various theories as to why boot has come to be used for boot meaning in the IT community. "

English word book for programmers (only)-Qiita
 [*](https://twitter.com/YumaInaura/status/1132296030421078017")

>"boot
>ブーツ（長靴）
>起動の意味は無い。なぜIT界隈だとbootが起動の意味に使われるようになったかは、諸説あるようだ。"
>
>プログラマ（だけ）のための英単語帳 - Qiita https://t.co/tCBzFCzXAk
# "default
Absence, default
The default value is the value when the specification was "absent", and originally there is no meaning of "initial" in the default. "

English word book for programmers (only)-Qiita
 [*](https://twitter.com/YumaInaura/status/1132296281454391297")

>"default
>欠席、不履行
>default valueは指定が「無かった」時の値であり、本来はdefaultに「初期」という意味はない。"
>
>プログラマ（だけ）のための英単語帳 - Qiita https://t.co/tCBzFCzXAk
